### PR TITLE
[A11y] Only show the VoiceOver notification once.

### PR DESCRIPTION
**Pull request by iainx**
Fixes VSTS #729402

### CODE REVIEW

**Comment by iainx** [comment]
@monojenkins backport release-8.2

**Comment by iainx** [comment]
@monojenkins backport release-8.1

**Comment by iainx** [comment]
Going to move the property to the Propertyservice instead of using NSUserDefaults

**Comment by mrward** [comment]
I was assuming NSUserDefaults would handle multiple instances running. The property service does not reload the settings after the IDE is started.

File: main/src/addins/MacPlatform/MacPlatform.cs [hunk 1]
```diff
@@ -370,10 +370,22 @@ static void NewInstanceMenuItem_Activated (object sender, EventArgs e)
...
 		const string EnabledKey = "com.monodevelop.AccessibilityEnabled";
+		const string VoiceOverNoticeShownKey = "com.monodevelop.VoiceOverShown";
 		static void ShowVoiceOverNotice ()
 		{
+			if (voiceOverNoticeShown || PropertyService.Get<bool> (VoiceOverNoticeShownKey, false)) {
+				return;
+			}
+
+			// Show the VoiceOver notice once
+			voiceOverNoticeShown = true;
```
Comment (mrward) [inline main/src/addins/MacPlatform/MacPlatform.cs]: Do we need to use both the PropertyService and the voiceOverNoticeShown? Seems like we could just use the PropertyService now.

File: main/src/addins/MacPlatform/MacPlatform.cs [hunk 2]
```diff
@@ -370,10 +370,22 @@ static void NewInstanceMenuItem_Activated (object sender, EventArgs e)
 				LoggingService.LogError ($"Failed to start new instance: {error.LocalizedDescription}");
 		}
 
-
+		// The Enabled key needs to be controlled through NSUserDefaults so that it can be
+		// set from the command line. This lets users who require accessibility features to
+		// enabled to control it through the commandline, if it has been disabled from inside the software
+		static bool voiceOverNoticeShown;
 		const string EnabledKey = "com.monodevelop.AccessibilityEnabled";
+		const string VoiceOverNoticeShownKey = "com.monodevelop.VoiceOverShown";
```
Comment (Therzok) [inline main/src/addins/MacPlatform/MacPlatform.cs]: Nit: Key naming style in propertyservice is like:
```suggestion
const string VoiceOverNoticeShownKey = "MonoDevelop.VoiceOver.Shown";
```

**Comment by sevoku** [comment]
@monojenkins backport release-8.2

**Pull request merged**